### PR TITLE
Added benchmark script

### PR DESCRIPTION
**Pull request by joel-mb**
This PR adds a new script to benchmark the leaderboard.

---
This change is [Reviewable](https://reviewable.io/reviews/carla-simulator/leaderboard/88)

### CODE REVIEW

File: scripts/leaderboard_benchmark_evaluator.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        else:
+            self.benchmark_file = open(args.benchmark_filename, "w")
+            serialize_header(self.benchmark_file)
+
+    def _register_statistics(self, config, checkpoint, entry_status, crash_message=""):
+        super(LeaderboardBenchmarkEvaluator, self)._register_statistics(
+            config, checkpoint, entry_status, crash_message
+        )
+
+        serialize_records(
```
Comment (glopezdiest) [inline scripts/leaderboard_benchmark_evaluator.py:79]: There is a weird interaction with the resume functionality in that crashed routes are repeated when restarting the simulation with the *resume* on. As such, that route is repeated at the records. I'd suggest checking if the crash_message is empty, to ensure that only the non-crashed routes are added to the records

Comment (joel-mb) [reply scripts/leaderboard_benchmark_evaluator.py]: Done

**Review by glopezdiest — APPROVED** [review]
> Reviewed 2 of 4 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/leaderboard/88)* status: 3 of 4 files reviewed, all discussions resolved (waiting on @joel-mb)

**Review by glopezdiest — APPROVED** [review]
> Reviewed 1 of 4 files at r1.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/leaderboard/88)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @joel-mb)

**Pull request closed without merging**